src/apps/loyalty/auth.ts

```diff
@@ -0,0 +1,7 @@
+export function login(email, pass, cb) {
+  cb(true)
```

Comment:
Yeah for sure, I think it'd probably makes sense to manage state in the loyalty page app. We can revisit that later though.